### ISSUE TITLE
On push receive - default push sound

### ISSUE THREAD

**Issue by jarekskuder** [arnesson/cordova-plugin-firebase#436]
There is no documentation on how to turn on sound when push comes. I know that it's possible, but don't know how. Help pls

**Comment by AsimNet** [comment]
https://github.com/arnesson/cordova-plugin-firebase/issues/315

**Comment by jarekskuder** [comment]
@AsimNet In firebase doc there is such thing - sound: default, would it work in cordova?

**Comment by AsimNet** [comment]
yes, but you have to add your custom sound manually in `res/raw/` folder in Android, same logic goes with iOS project..

[https://github.com/arnesson/cordova-plugin-firebase/issues/315#issuecomment-309630913](link)
@jarekskuder

**Comment by jarekskuder** [comment]
@AsimNet So what you are saying that if i send with push sound: default i still need to place sound file to res/raw/ folder ?

**Comment by AsimNet** [comment]
@jarekskuder no, I don't think so, yet I didn't test it !

**Comment by jarekskuder** [comment]
Well @AsimNet i went to java code in this plugin and found out that there is no need of some sound parameter at all, i don't see logic that there is a need of sound default param, also i added the sound logic when push comes in foreground, but still no luck with sound when in background. I tried everything in background logic, but no luck (well i'm not the best java programmer, but still...). So the problem stays, no sound when app in background

**Comment by jarekskuder** [comment]
#458 my new issue
